manifests/services/application/deplyoment.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: managed-gitops
+spec:
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: managed-gitops
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: managed-gitops
+    spec:
+      containers:
+      - name: managed-gitops
+        image: quay.io/samyakjain/managed-gitops:latest
```

Comment:
I setup a new repo for us to use, and sent you an invite: https://quay.io/repository/redhat-appstudio/gitops-service?tab=tags

See if you can push to there, then this line can be updated to `quay.io/redhat-appstudio/gitops-service:latest`

Reply:
Got the invite, working, thanks! :dango: